v1.2.yml

```diff
@@ -1608,3 +1784,259 @@ definitions:
       email:
         type: string
         x-nullable: true
+
+  StudentObject:
```

Comment:
Yes, it is possible. I'm not sure if readme.io support it, and I know wag doesn't, so I've been avoiding doing it for now. That said, if we keep having to update these in multiple places we should look into it more.